apps/site/assets/css/_modal-component.scss

```diff
@@ -37,6 +37,10 @@
     transform: translate(-50%, -50%);
     width: 75%;
   }
+
+  @include media-breakpoint-up(lg) {
+    width: 55%;
```

Comment:
Let's add a max-width of 40rem

Reply:
Added max-width. 
I went for 55% as ~ 7 col / 12 col.